spec/p2p/v0.34/README.md

```diff
@@ -52,18 +52,18 @@ which implements the following abstractions:
 
 Existing documentation referring to the p2p layer:
 
-- <https://github.com/cometbft/cometbft/tree/main/spec/p2p>: p2p-related
+- <https://github.com/cometbft/cometbft/tree/v0.38.x/spec/p2p>: p2p-related
```

Comment:
should we make these links relative, so they get locked by branch automatically?

Reply:
I think we need a bigger effort to make as many of the docs' links relative as we can.